charts/external-dns/README.md

```diff
@@ -74,3 +74,4 @@ The following table lists the configurable parameters of the _ExternalDNS_ chart
 | `secretConfiguration.enabled`     | Enable additional secret configuration.                                                                                                                                                                                                                                                                               | `false`                                |
 | `secretConfiguration.mountPath`   | Mount path of secret configuration secret (this can be templated).                                                                                                                                                                                                                                                    | `""`                                   |
 | `secretConfiguration.data`        | Secret configuration secret data. Could be used to store DNS provider credentials.                                                                                                                                                                                                                                    | `{}`                                   |
+| `secretConfiguration.data`        | Additional secret configuration. Can be used to store dns provider credentials                                                                                                                                                                                                                                        | `{type: Recreate}`                     |
```

Comment:
```suggestion
| `secretConfiguration.subPath`        | Sub-path of secret configuration secret (this can be templated).                                                                                                                                                                                                                                        | `""`                     |
```

Reply:
seems like I had a brain lag here. sorry.